.gitignore

```diff
@@ -141,6 +142,3 @@ Desktop.ini
 # ESLint
 ######################
 .eslintcache
-
-src/main/lep-sandbox
```

Comment:
please keep this directory ignored to connect LEP scripts with symlink.

Reply:
done